docs/public/installation/prerequisites.md

```diff
@@ -77,12 +81,12 @@ The latest version of the browser at the time of testing is used.
 Operating System | Browser
 ---------------- | -------
 Debian 12        | Firefox
-MacOS 13         | Firefox
-MacOS 13         | Chrome
-Windows 10       | Firefox
-Windows 10       | Chrome
-Ubuntu 22.04     | Firefox
-Ubuntu 22.04     | Chrome
+FreeBSD 14.1     | ?
+Rocky Linux 8.10 | ?
+Rocky Linux 9.4  | ?
+Ubuntu 22.04     | ?
+Ubuntu 22.04     | ?
+Ubuntu 24.04     | ?
```

Comment:
I agree. Since we do not do any thorough tests anyway we might as well just write that we test Zonemaster in both Firefox and Chrome. I do that on MacOS. I could do it on Windows and Ubuntu as well, but not on FreeBSD. We have never had any issues connected to which OS that the Browser runs on. I think we have seen issues connected to Firefox or Chrome.